src/data/processors/phoneNumber.ts

```diff
@@ -0,0 +1,32 @@
+import type { ParagraphPhoneNumber } from '@/types/drupal/paragraph'
+import { FormattingError } from '../errors/formatting'
+
+export type PhoneNumber = {
+  extension?: string
+  label?: string
+  number: string
+  type?: string
+}
+
+export function processPhoneNumber(
+  phoneNumberParagraph: ParagraphPhoneNumber
```

Comment:
This does not handle Phone number non-paragraphs... which we still have around Drupal but not here or in any of the content added to next-build so far. So we may want to expand on this when we add that type of content.